### PR TITLE
[7.3.x] RHBPMS-4942 - LazyInitializationException when getting process variab…

### PR DESCRIPTION
**Pull request by tsurdilo**
…les which extends VariableEntity

### CODE REVIEW

**Comment by tsurdilo** [comment]
depends on drools pr : https://github.com/kiegroup/drools/pull/1520

**Comment by tsurdilo** [comment]
@MarianMacik updated according to your comments - thanks!

**Comment by tsurdilo** [comment]
@mswiderski updated like the master pr - thanks!

**Comment by tsurdilo** [comment]
Jenkins retest please

File: jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/ProcessServiceWithEntitiesTest.java [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        logger.debug("Preparing kjar");
+        KieServices ks = KieServices.Factory.get();
+        ReleaseId releaseId = ks.newReleaseId(GROUP_ID, ARTIFACT_ID, VERSION);
+        List<String> processes = new ArrayList<String>();
+        processes.add("repo/processes/general/entityprocessvar-process.bpmn2");
+
+
+        DeploymentDescriptorImpl customDescriptor = new DeploymentDescriptorImpl("org.jbpm.domain");
+        customDescriptor.setAuditMode(AuditMode.JPA);
+        customDescriptor.setEnvironmentEntries(null);
```
Comment (mswiderski) [inline jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/ProcessServiceWithEntitiesTest.java]: for consistency we could remove these two

**Comment by mswiderski** [comment]
and I guess we should move this to 7.4.x as 7.3.x is no longer under development

**Comment by mswiderski** [comment]
we should move this to 7.4.x

**Comment by tsurdilo** [comment]
@mswiderski removing as created 7.4.x pr. Thanks.

**Pull request closed without merging**